packages/palette_generator/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0-nullsafety.0
```

Comment:
Once https://github.com/flutter/packages/pull/289 lands this can just be 0.3.0 so we don't need to do a second pass over this package.